package.template.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "project-name",
```

Comment:
yes very nice setup script. However we still need to mention it in the readme right?